keadm/cmd/keadm/app/cmd/deprecated/init.go

```diff
@@ -130,6 +130,13 @@ func Add2CloudToolsList(toolList map[string]types.ToolsInstaller, flagData map[s
 			fmt.Println("Failed to get the latest KubeEdge release version, will use default version: ", kubeVer)
 		}
 	}
+
+	// If users specified the kubeedge-version with prefix `v`, remove it
+	// So we support both format: 1.12.0 without prefix `v`, and v1.12.0 with prefix `v`
+	if strings.HasPrefix(kubeVer, "v") {
```

Comment:
strings.TrimPrefix will call the function strings.HasPrefix and it's no need to repeatedly call it here.